doc/TESTS.md

```diff
@@ -20,13 +20,20 @@ rake test
 
 These tests use [rspec](http://rspec.info/) are not executed on travis, so you need to launch the execution manually.
 
-You can use the following command:
+
+## Run your tests always on your forked repo!
```

Comment:
This shouldn't be a title but something like:

```WARNING```: Run your tests always on your forked repo!

Reply:
ok